app/validators/url_validator.rb

```diff
@@ -0,0 +1,12 @@
+class UrlValidator < ActiveModel::EachValidator
+  def validate_each(record, attribute, value)
```

Comment:
forgot to mention this in pr, this class is a custom validator for the url of thumbnails so that thumbnails cannot be created without a real url